.gitignore

```diff
@@ -6,3 +6,6 @@
 /esdoc
 /node_modules
 /demo/kinto*
+/.vscode
+.DS_Store
```

Comment:
Hmm 🤔  I believe that most people (like me for example) have a global `.gitconfig` and only a per-project config. I can remove that if you want?